addons/stock_landed_costs/models/account_move.py

```diff
@@ -73,3 +73,11 @@ def _onchange_is_landed_costs_line_product(self):
             self.is_landed_costs_line = True
         else:
             self.is_landed_costs_line = False
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        lines = super().create(vals_list)
+        # apply correct account to landed costs lines
+        for line in lines.filtered('is_landed_costs_line'):
+            line._onchange_is_landed_costs_line()
```

Comment:
Not a big fan of calling an onchange method. Mainly because of the potential performance issue. Would be nice to extract the essence of the onchange function. Make it "multi" and call it in create as well as in the onchange.